docs/content/docs/guide/incoming_webhook.md

```diff
@@ -70,7 +70,13 @@ after setting this up, you will be able to trigger a PipelineRun called
 `https://github.com/owner/repo`. As an example here is the full curl snippet:
 
 ```shell
-curl -X POST https://control.pac.url/incoming?secret=very-secure-secret&repository=repo,branch=main&pipelinerun=target_pipelinerun
+curl -X POST https://control.pac.url/incoming?secret=very-secure-shared-secret&repository=repo,branch=main&pipelinerun=target_pipelinerun
+```
+
+OR
```

Comment:
i don't think you need this, just change the original example quoted with single quote `'`